cookiecutter/prompt.py

```diff
@@ -183,8 +183,11 @@ def prompt_for_config(context, no_input=False):
     :param dict context: Source for field names and sample values.
     :param no_input: Prompt the user at command line for manual configuration?
     """
+    conditional_keyword = "_if_use_"
```

Comment:
Since `_` already has a meaning in cookiecutter identifiers, I would tend to use a different syntax here.
Ideas: 
- `[if use VARNAME]`
- `? use VARNAME`

(more ideas?

Reply:
As currently it is vatiable, I think we can leave it as it. For now. naming is not my best )

Reply:
Hi @jensens, great suggestion! I can see what I can do about the naming issue. For example, as a third option, I could introduce ___ (3x _) as a prefix. I will try and see!

Reply:
@nilsdebruin any update on this?  I'd like to see this land as I have a need for the conditional questions as well